### ISSUE TITLE
Kubernetes-cluster wiring

### ISSUE THREAD

**Issue by fieldhof** [bpetri/nativeLoad#33]
When starting the kubernetes-cluster-demo with only the Celix queue and without the viewer the Queue doesn't get registered in etcd.

**Comment by fieldhof** [comment]
When manualy starting celix within the docker image I get this error:

```
ERROR: Bundle exception [70001]: "Could not load library: .cache/bundle5/version0.1/liborg.inaetics.remote_service_admin.so";  
 Cause: libdfi.so: cannot open shared object file: No such file or directory
    at framework_loadLibrary(/tmp/celix/framework/private/src/framework.c:2475)  
ERROR: Bundle exception [70001]: "Could not load all libraries"
    at framework_loadLibraries(/tmp/celix/framework/private/src/framework.c:2423)  
ERROR: Bundle exception [70001]: "Could not load all bundle libraries"
    at framework_loadBundleLibraries(/tmp/celix/framework/private/src/framework.c:2377)  
ERROR: Bundle exception [70001]: Could not start bundle: (null) [5]
    at framework_markBundleResolved(/tmp/celix/framework/private/src/framework.c:1773)
```

Way to start celix:
- Ssh into the correct node `vagrant ssh node-x`
- Check the id of the container `docker ps`
- Get into the docker container `docker exec -t -i {ID} /bin/bash`
- Go to the workdir `cd /tmp/workdir/`
- Start celix `celix config.properties`

**Comment by fieldhof** [comment]
Fixed in commit d8e8fab.
